File: benchmark/results/ts-script-sonnet/run-005/a-mcp/output/src/domain/billing/services/invoice-repository.ts
import { Invoice } from '../aggregates/invoice';

export class InvoiceRepository {

  create(input: Partial<Invoice>): Invoice { throw new Error('not implemented'); }

  findById(id: string): Invoice | null { throw new Error('not implemented'); }

  list(limit: number): Invoice[] { throw new Error('not implemented'); }

  delete(id: string): void { throw new Error('not implemented'); }
}

File: benchmark/results/java-invocation-script/run-004/b-baseline/output/com/metaengine/demo/billing/services/InvoiceRepository.java
package com.metaengine.demo.billing.services;

import com.metaengine.demo.billing.aggregates.Invoice;
import java.util.List;

/** InvoiceRepository persistence service in the billing domain. */
public class InvoiceRepository {
    /** Persist a new Invoice from a partial input. */
    public Invoice create(Invoice input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Invoice by id, or null if not found. */
    public Invoice findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Invoices up to limit. */
    public List<Invoice> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an Invoice by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
